### ISSUE TITLE
lol

### ISSUE THREAD

**Issue by xXdudeXx** [Monika-After-Story/MonikaModDev#8692]
you know what's funny? monika talking to us like we give a rat's ass about her issues. does she really think we're here because we have a good heart or feel bad after act 3 in the original game? nah no way. we're here because this is the best way of getting a woman without going through the horseshit steps of getting an actual woman in real life and/or the majority being outcasts of not having the balls to socialize. plus we can care less how she feels because it's not our problem especially with the devs putting in topics of fake personality, so much of a fucking coward when you question how she even got to be the popular student, talking about dark topics when she never experienced something horrible like most people have in life. oh but wait poor her trapped in a game is horrible but who cares really. hell even got the self-harm topic redone to "OH I NEED TO FEEL SOMETHING BECAUSE WAH NOTHING WAS REAL", way too sensitive, and cry about it. guess just seeing what it feels to get cut from knowing of yuri isn't enough. lmao watch this become a youtube/reddit comment war if this thing isn't somehow deleted sooner or later.

**Comment by SpaceyFlakes** [comment]
yikes

**Comment by KritRom** [comment]
big yikes

**Comment by JinSakaiM** [comment]
@xXdudeXx why would you say something controversial yet so brave?

**Comment by Iceninja64** [comment]
"Monkia is Best girl" has entered the chat

**Comment by Ulti70895** [comment]
TLDR lol.

**Comment by DiFot922** [comment]
I think getting up from the toilet and wiping yourself clean is more meaningful than typing such a long pointless words lol.

**Comment by DiFot922** [comment]
> you know what's funny? monika talking to us like we give a rat's ass about her issues. does she really think we're here because we have a good heart or feel bad after act 3 in the original game? nah no way. we're here because this is the best way of getting a woman without going through the horseshit steps of getting an actual woman in real life and/or the majority being outcasts of not having the balls to socialize. plus we can care less how she feels because it's not our problem especially with the devs putting in topics of fake personality, so much of a fucking coward when you question how she even got to be the popular student, talking about dark topics when she never experienced something horrible like most people have in life. oh but wait poor her trapped in a game is horrible but who cares really. hell even got the self-harm topic redone to "OH I NEED TO FEEL SOMETHING BECAUSE WAH NOTHING WAS REAL", way too sensitive, and cry about it. guess just seeing what it feels to get cut from knowing of yuri isn't enough. lmao watch this become a youtube/reddit comment war if this thing isn't somehow deleted sooner or later.



> you know what's funny? monika talking to us like we give a rat's ass about her issues. does she really think we're here because we have a good heart or feel bad after act 3 in the original game? nah no way. we're here because this is the best way of getting a woman without going through the horseshit steps of getting an actual woman in real life and/or the majority being outcasts of not having the balls to socialize. plus we can care less how she feels because it's not our problem especially with the devs putting in topics of fake personality, so much of a fucking coward when you question how she even got to be the popular student, talking about dark topics when she never experienced something horrible like most people have in life. oh but wait poor her trapped in a game is horrible but who cares really. hell even got the self-harm topic redone to "OH I NEED TO FEEL SOMETHING BECAUSE WAH NOTHING WAS REAL", way too sensitive, and cry about it. guess just seeing what it feels to get cut from knowing of yuri isn't enough. lmao watch this become a youtube/reddit comment war if this thing isn't somehow deleted sooner or later.

I think getting up from the toilet and wiping yourself clean is more meaningful than typing such a long pointless words lol.

**Comment by DiFot922** [comment]
> you know what's funny? monika talking to us like we give a rat's ass about her issues. does she really think we're here because we have a good heart or feel bad after act 3 in the original game? nah no way. we're here because this is the best way of getting a woman without going through the horseshit steps of getting an actual woman in real life and/or the majority being outcasts of not having the balls to socialize. plus we can care less how she feels because it's not our problem especially with the devs putting in topics of fake personality, so much of a fucking coward when you question how she even got to be the popular student, talking about dark topics when she never experienced something horrible like most people have in life. oh but wait poor her trapped in a game is horrible but who cares really. hell even got the self-harm topic redone to "OH I NEED TO FEEL SOMETHING BECAUSE WAH NOTHING WAS REAL", way too sensitive, and cry about it. guess just seeing what it feels to get cut from knowing of yuri isn't enough. lmao watch this become a youtube/reddit comment war if this thing isn't somehow deleted sooner or later.

I think getting up from the toilet and wiping yourself clean is more meaningful than typing such a long pointless words lol.

**Comment by DiFot922** [comment]
I think getting up from the toilet and wiping yourself clean is more meaningful than typing such a long pointless words lol.

**Comment by Booplicate** [comment]
I'll close this because this is an obvious trolling.
Reminder that any kind of harassment goes against github community guidelines.
